public/app/themes/justice/src/sass/admin.scss

```diff
@@ -45,3 +47,8 @@
 div.cms_tpv_promo_above_wrapper {
     display: none !important;
 }
+
+// Recolour edit banner to better fit the theme
+.rvy_preview_msgspan {
+  background-color: #373737 !important;
+}
```

Comment:
Love this @EmilyHazlehurst - to be certain; will this file display on the front end if it's an admin style?

Reply:
@wilson1000 this was moved to the app.scss file, you must have commented as it was being pushed. That should be working now :)